frontend-react/.env.test

```diff
@@ -1,13 +1,13 @@
-;; this is set by env var being set `CLIENT_ENV: test` in the github action
+; this is set by env var being set `CLIENT_ENV: test` in the github action
 REACT_APP_TITLE=CDC Prime ReportStream TEST
 REACT_APP_DESCRIPTION=ReportStream Dashboard Application
-REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+REACT_APP_ICON=https://test.reportstream.cdc.gov//assets/cdc-logo.svg
```

Comment:
Why `.gov//` instead of just `.gov/`? I see it many places so I assume this is an intentional thing that I'm unaware of!

Reply:
fixed